src/app/cli/src/commands/log_command.rs

```diff
@@ -343,25 +343,14 @@ impl AsciiRenderer {
                 }
                 self.render_property(output, 0, "WebsiteURL", &e.website_url)?;
             }
-            MetadataEvent::SetPollingSource(SetPollingSource {
-                fetch: _,
-                prepare: _,
-                read: _,
-                preprocess: _,
-                merge: _,
```

Comment:
In this file I was specifying all fields on purpose to force whoever is changing metadata to consider this command. The `..` pattern is more like "don't care and won't ever care", so exhaustive match has its uses.

Reply:
Looking at the statement body, it does not look like it was supposed to be ever extended.